### DISCUSSION TITLE
Where is the UNESCO-listed Forbidden City located?

### DISCUSSION THREAD

**Discussion by ParanoidUser** [ParanoidUser/yolo#642] [Galaxy Trivia] [answered]
**Bonus:** Why is it named that way?

**Comment by DevElkami** [comment]
The Forbidden City are located in the centre of Beijing.

It was so named because access to the area was barred to most of the subjects of the realm. Government functionaries and even the imperial family were permitted only limited access; the emperor alone could enter any section at will.

**Comment by peterbaccaro** [comment, marked as answer]
The UNESCO World Heritage Site known as the Forbidden City (also known as the Palace Museum) is located in **Beijing (China)**.
It was named the Forbidden City because it was the imperial palace of the Ming and Qing dynasties from the 15th to the early 20th century and it was forbidden for anyone to enter or leave the palace without the emperor's permission.  Only the imperial family, court officials and selected servants were allowed to enter the palace complex. The name Forbidden City has been used since the 19th century when it was opened to the public and **refers to the fact that the palace was off limits to most people for many centuries**.
